docs/examples/runai-streamer/playground-streaming-from-s3.yaml

```diff
@@ -0,0 +1,54 @@
+# This example demonstrates how to use the Run:ai Model Streamer to load models directly from S3.
+# Adding the annotation `llmaz.io/skip-model-loader: "true"` skips the model-loader initContainer,
+# allowing the inference engine to load models directly from remote storage (e.g., S3).
+# By using `--load-format runai_streamer`, the vLLM leverages the Run:ai Model Streamer to stream models from S3.
+# vLLM will load models into the CPU buffer and then into GPU memory, without the need to download them to local disk first.
+# This can significantly improve model loading speed and reduce disk usage.
+apiVersion: llmaz.io/v1alpha1
+kind: OpenModel
+metadata:
+  name: deepseek-r1-distill-qwen-1-5b
+  annotations:
```

Comment:
Can we controller this for Playground and iSVC, I think it's more flexible there.

Reply:
But we can follow this later ofcourse.

Reply:
I personally don't think this is a typical example, load different models with different approaches.

Reply:
Ok, I'll refactor it tomorrow.